botorch/optim/parameter_constraints.py

```diff
@@ -303,14 +303,14 @@ def _generate_unfixed_lin_constraints(
                 new_coefficients.append(coefficient)
             # otherwise, we "remove" the constraints corresponding to that index
             else:
-                new_rhs -= coefficient.item() * ffval_or_None
+                new_rhs = new_rhs - coefficient.item() * ffval_or_None
```

Comment:
Thank you for fixing this so quickly. I think `rhs` might have been of the wrong type in the bug report causing this.
Nonetheless, this is a harmless change.

Reply:
Yeah I think ideally we can support both python scalars and single-element tensors.